mainsail/docker-compose.yml

```
@@ -0,0 +1,13 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: mainsail_web_1
+      APP_PORT: 80
+  web:
+    image: ghcr.io/mainsail-crew/mainsail:sha-4c86d7c@sha256:960b63d7f1c438ab68b0add4a5eac0aa972ec29b13e3ec16ccbc2dfadf86d568
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/config.json:/usr/share/nginx/html/config.json:ro
```

Change to data directory after creating it.
```suggestion
      - ${APP_DATA_DIR}/data/config.json:/usr/share/nginx/html/config.json:ro
```